types/win-ca/index.d.ts

```diff
@@ -0,0 +1,64 @@
+// Type definitions for win-ca 3.5
+// Project: https://github.com/ukoloff/win-ca
+// Definitions by: Marcos Junior <https://github.com/junalmeida>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+import { pki } from "node-forge";
+
+declare namespace Api {
```

Comment:
please stick to native mdoule symbols and use 'api' symbol name, works fine. 'api' is not a symbol like Class, so I think there is no need to use different notation that can mislead users.